lte/gateway/c/core/oai/test/spgw_task/spgw_test_util.cpp

```diff
@@ -42,8 +43,12 @@ bool is_num_sessions_valid(
 
   // If one UE context exists, check that teids also exist in hashtable
   spgw_ue_context_t* ue_context_p = spgw_get_ue_context(imsi64);
-  int num_teids                   = 0;
-  sgw_s11_teid_t* s11_teid_p      = nullptr;
+  if (expected_num_teids == 0) {
```

Comment:
can you add a comment to explain the if case here?

Reply:
@thhung ,  we need two different validation functions here:
-- `is_ue_teids_valid(unit64_t imsi64, int expected_num_ue_teids)` to check if UE context is non-null and tunnel count is valid for a single UE
-- `is_num_sessions_valid(int expected_num_ue_contexts)` to check conditions for multiple UEs 

Note that on latest master, I have removed the `spgw_state` parameter from this function as it is unused